app/common/lib/shieldUtil.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const frameStateUtil = require('../../../js/state/frameStateUtil')
```

Comment:
Yeah I wanted to create util for now and then when moving to redux create shield state. What do you think should we move this into frame or shield state? Keep in mind that we have a lot of logic for the panel